src/radical/pilot/configs/resource_aliases.json

```diff
@@ -1,33 +1,14 @@
 
 {
     "aliases": {
-        "das4.fs2"                          : "das4.fs2_ssh",
-        "das5.fs1"                          : "das5.fs1_ssh",
-        "epsrc.archer"                      : "epsrc.archer_orte",
         "fub.allegro"                       : "fub.allegro_rsh",
-        "futuregrid.india"                  : "futuregrid.india_ssh",
-        "futuregrid.bravo"                  : "futuregrid.bravo_ssh",
-        "futuregrid.delta"                  : "futuregrid.delta_ssh",
-        "futuregrid.echo"                   : "futuregrid.echo_ssh",
-        "futuregrid.xray"                   : "futuregrid.xray_aprun",
-        "iu.bigred2"                        : "iu.bigred2_aprun",
-        "lrz.supermuc"                      : "lrz.supermuc_ssh",
-        "lumc.shark"                        : "lumc.shark_ssh",
-        "lumc.gb-ui"                        : "lumc.gb-ui_ssh",
         "ncar.yellowstone"                  : "ncar.yellowstone_ssh",
         "ncsa.bw"                           : "ncsa.bw_orte",
```

Comment:
BW will be back as a cluster dedicated to climate image processing. Hopefully, its stack will change but even if it returns as it was, should we have bw_prrte?

Reply:
We should add a valid config once we can test it, but its probably not worthwhile including a config which we know won't work... :/  I'll remove the BW configs for now, ok?